tutor/src/models/task-plans/teacher/plan.js

```diff
@@ -22,6 +22,25 @@ const TUTOR_SELECTIONS = {
 };
 
 
+const calculateDefaultOpensAt = ({ course }) => {
+  const defaultOpensAt = moment(Time.now).add(1, 'day').startOf('minute');
+  if (!course) {
+    return defaultOpensAt.toISOString();
+  }
+  const [ hour, minute ] = course.default_open_time.split(':');
+
+  return moment(
+    findLatest(
+      findEarliest(
+        course.bounds.end,
+        defaultOpensAt,
+      ),
+      course.bounds.start.add(1, 'minute'),
+    ),
+  ).hour(hour).minute(minute).startOf('minute').toISOString();
```

Comment:
I think the `hour` and `minute` need to be applied directly to `defaultOpensAt` (before the comparison). Otherwise subtract a day from `course.bounds.end` and add a day to `course.bounds.start` (instead of 1 minute).

Reply:
Hm, I think course start/end is always midnight, but yah wouldn't hurt to add a day to be safe.  i'll do that